doc/en/goodpractises.rst

```diff
@@ -306,32 +242,40 @@ using the ``--pytest-args`` or ``-a`` command-line option. For example::
 
 is equivalent to running ``py.test --durations=5``.
 
-.. seealso::
 
-    For a more powerful solution, take a look at the
-    `pytest-runner <https://pypi.python.org/pypi/pytest-runner>`_ plugin.
+.. _standalone:
+.. _`genscript method`:
 
-.. _`test discovery`:
-.. _`Python test discovery`:
+(deprecated) Create a pytest standalone script
+-----------------------------------------------
 
-Conventions for Python test discovery
--------------------------------------------------
+.. deprecated:: 2.8
 
-``pytest`` implements the following standard test discovery:
+.. note::
 
-* collection starts from paths specified in :confval:`testpaths` if configured,
-  otherwise from initial command line arguments which may be directories,
-  filenames or test ids. If :confval:`testpaths` is not configured and no
-  directories or files were given in the command line, start collection from
-  the current directory.
-* recurse into directories, unless they match :confval:`norecursedirs`
-* ``test_*.py`` or ``*_test.py`` files, imported by their `test package name`_.
-* ``Test`` prefixed test classes (without an ``__init__`` method)
-* ``test_`` prefixed test functions or methods are test items
+    ``genscript`` has been deprecated because:
 
-For examples of how to customize your test discovery :doc:`example/pythoncollection`.
+    * It cannot support plugins, rendering its usefulness extremely limited;
+    * Tooling has become much better since ``genscript`` was introduced;
+    * It is possible to build a zipped ``pytest`` application without the
+      shortcomings above (this will be documented in more detail in a later
+      version of this documentation).
```

Comment:
I took these points from @RonnyPfannschmidt in [this comment](https://github.com/pytest-dev/pytest/pull/884#issuecomment-159566507)... Do you have a link for that @RonnyPfannschmidt? Otherwise we might just add this later.